zerver/lib/actions.py

```diff
@@ -2659,6 +2659,27 @@ def truncate_topic(topic):
     # type: (Text) -> Text
     return truncate_content(topic, MAX_SUBJECT_LENGTH, "...")
 
+def do_update_buddy_list(user_profile, buddy_profile, should_add):
+    # type: (UserProfile, str, bool) -> None
+
+    count = BuddyList.objects.filter(user=user_profile, buddy=buddy_profile).count()
+    if count > 0:
+        if should_add == False:
+            BuddyList.objects.filter(user=user_profile, buddy=buddy_profile).delete()
+    else:
+        if should_add:
+            BuddyList.objects.create(user=user_profile, buddy=buddy_profile)
+
+
+def get_buddy_list(user_profile):
+    # type: (UserProfile) -> str
+
+    records = BuddyList.objects.filter(user=user_profile)
+    user_list = []
+    for record in records:
+        user_list.append(record.buddy.email)
+
+    return ",".join(user_list)
```

Comment:
It probably makes sense to send this to the frontend in list form, rather than as a string joined by commas.  Can you change that?